### ISSUE TITLE
add option to enable regular hls

### ISSUE THREAD

**Issue by darkdrgn2k** [tomeshnet/ipfs-live-streaming#55]
on rtmp or maybe optional new server 

code to activate 
https://github.com/tomeshnet/ipfs-live-streaming/commit/acd4eb5dc257ce4468cb475a4ba22475410e66e7

**Comment by benhylau** [comment]
Yeah probably new server. RTMP path would allow pushing to YouTube and stuff, but that doesn't really integrate with our embedded player UI. I think we can just do a new server and if people want to push to their own cluster like @ASoTNetworks for Our Networks it's trivial enough to change this code.
